tests/test_validate_resources.py

```diff
@@ -2,6 +2,20 @@
 import pytest
 
 from tests.scripts.validate_resources import parse_resource_file_for_errors, resource_file
+from ocp_resources.resource import replace_key_with_hashed_value
```

Comment:
please add negative tests as well

Reply:
I have added some of the negative tests.